modules/transactions/src/testFixtures/java/org/apache/ignite/internal/tx/storage/state/AbstractTxStateStorageTest.java

```diff
@@ -75,8 +74,7 @@ void beforeTest() {
         tableStorage.start();
     }
 
-    @AfterEach
-    void afterTest() {
+    protected void afterTest() {
```

Comment:
Why did you remove `@AfterEach` here?

Reply:
I tried to use the annotation `@Override` and `@AfterEach` at the same time and found that none of them is called, I read [here](https://stackoverflow.com/questions/48864716/before-beforeeach-inheritance-behaviour-change-junit4-junit5) that in JUnit5 you need to do it differently than in JUnit4, I did it.

Reply:
fix it